yapapi/services/service.py

```diff
@@ -356,7 +356,7 @@ async def shutdown(self):
         yield s
 
     async def reset(self) -> None:
-        """Reset the service to the initial state.
+        """Reset the service to the initial state. Default implementation is no operation
 
         This method is called internally when the service is restarted in :class:`yapapi.services.ServiceRunner`,
         so it is not necessary for services that are never restarted (note that :func:`~yapapi.Golem.run_service()` by
```

Comment:
we should also mention on line 363 that the default is controlled by `restart_condition`